test/runtests.jl

```diff
@@ -149,13 +149,14 @@ end
     obs = Observable(5)
     @test string(obs) == "Observable(5)"
     f = on(identity, obs)
-    @test occursin("Observable(5)\n    0 => identity(x) in Base at operators.jl", plain(obs))
+    @test occursin("Observable(5)\n    0 => identity(x)", plain(obs))
     @test string(f) == "ObserverFunction `identity` operating on Observable(5)"
     f = on(x->nothing, obs); ln = @__LINE__
     str = plain(obs)
     @test occursin("Observable(5)", str)
-    @test occursin("0 => identity(x) in Base at operators.jl", str)
-    @test occursin(" in Main at $(@__FILE__)", str)
+    @test occursin("0 => identity(x)", str)
+    @test occursin(" Main", str)
+    @test occursin("runtests.jl", str)
```

Comment:
I had to weaken these tests because the printing of this sort of thing has changed on v1.11, and it was causing the tests to fail.